functions/datastore/test/index.test.js

```diff
@@ -33,178 +44,270 @@ const VALUE = {
-    .post('/get')
-    .send({
-      kind: KIND,
-      key: NAME,
-    })
-    .expect(200)
-    .expect(response => {
-      assert.deepStrictEqual(JSON.parse(response.text), {
+    it('get: Finds an entity', async () => {
+      const response = await requestRetry({
+        method: 'POST',
+        url: `${BASE_URL}/get`,
+        body: {
+          kind: KIND,
+          key: NAME,
+        },
+        json: true,
+      });
+
+      assert.strictEqual(response.statusCode, 200);
+      assert.deepStrictEqual(response.body, {
         description: 'Buy milk',
       });
     });
-});
 
-it('del: Fails without a key', () => {
-  const req = {
-    body: {},
-  };
-  assert.throws(
-    () => {
-      program.del(req, null);
-    },
-    Error,
-    errorMsg('Kind')
-  );
-});
+    it('get: Fails without a key', async () => {
+      const req = {
+        body: {},
+      };
 
-it('del: Fails without a kind', () => {
-  const req = {
-    body: {
-      key: NAME,
-    },
-  };
-  assert.throws(
-    () => {
-      program.del(req, null);
-    },
-    Error,
-    errorMsg('Kind')
-  );
-});
+      const res = {
+        status: sinon.stub().returnsThis(),
+        send: sinon.stub(),
+      };
+
+      await program.get(req, res);
 
-// TODO: ace-n Figure out why this test started failing
-it.skip(`del: Doesn't fail when entity does not exist`, async () => {
-  await request
-    .post('/del')
-    .send({
-      kind: KIND,
-      key: 'nonexistent',
-    })
-    .expect(200)
-    .expect(response => {
-      assert.strictEqual(response.text, `Entity ${KIND}/nonexistent deleted.`);
+      assert.ok(res.status.calledWith(500));
+      assert.ok(res.send.calledWith(errorMsg('Key')));
     });
-});
 
-// TODO: ace-n Figure out why this test started failing
-it.skip('del: Deletes an entity', async () => {
-  await request
-    .post(`/del`)
-    .send({
-      kind: KIND,
-      key: NAME,
-    })
-    .expect(200)
-    .expect(response => {
-      assert.strictEqual(response.text, `Entity ${KIND}/${NAME} deleted.`);
-    });
-
-  const key = datastore.key([KIND, NAME]);
-  const [entity] = await datastore.get(key);
-  assert.ok(!entity);
+    it('get: Fails without a kind', async () => {
+      const req = {
+        body: {
+          key: NAME,
+        },
+      };
+
+      const res = {
+        status: sinon.stub().returnsThis(),
+        send: sinon.stub(),
+      };
+
+      await program.get(req, res);
+
+      assert.ok(res.status.calledWith(500));
+      assert.ok(res.send.calledWith(errorMsg('Kind')));
+    });
+  });
+
+  describe('del', () => {
+    let ffProc;
+
+    before(() => {
+      ffProc = execPromise(
+        `functions-framework --target=del --signature-type=http`,
+        {timeout: FF_TIMEOUT, shell: true, cwd}
+      );
+    });
+
+    after(async () => {
+      await handleLinuxFailures(ffProc);
+    });
+
+    it('del: Fails without a key', async () => {
+      const req = {
+        body: {},
+      };
+
```

Comment:
Nit: Remove unnecessary \n. We can put `const` declarations together. Even putting the `const` declarations next to the `assert` statements looks fine like `del: Deletes an entity`.